e2e/kots-release-install/cluster-config.yaml

```diff
@@ -40,6 +40,11 @@ spec:
         spec:
           telemetry:
             enabled: false
+          workerProfiles:
+          - name: ip-forward
+            values:
+              allowedUnsafeSysctls:
+              - net.ipv4.ip_forward
```

Comment:
I would prefer that we add this to the `-upgrade` and `-install-noop` directories too